ansible/roles/slurm_exporter/tasks/install.yml

```diff
@@ -0,0 +1,72 @@
+- name: Create slurm exporter group
+  ansible.builtin.group:
+    name: "{{ slurm_exporter_group }}"
+    state: present
+    system: true
+
+- name: Create user
+  ansible.builtin.user:
+    name: "{{ slurm_exporter_user }}"
+    groups: "{{ slurm_exporter_group }}"
+    append: true
+    shell: /usr/sbin/nologin
+    system: true
+    create_home: false
+    home: /
+
+- name: Install GCC
+  ansible.builtin.dnf:
+    name: gcc
+    state: latest
```

Comment:
Might `latest` upgrade gcc unpredictably? Probably doesn't matter on a control host, but people might be annoyed if it was a login node

Reply:
Agreed, good catch